crowbar_framework/vendor/assets/javascripts/jquery/spritely.js

```diff
@@ -1,539 +0,0 @@
-/*
```

Comment:
@tboerger my comment was really to avoid any risk for the release; since this won't make it in the release (but for an update), I'm fine removing the file now.